compendium/test_bundles/TestBundleMultipleManagedService/src/ManagedServiceImpl.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ManagedServiceImpl.hpp"
+
+#include <iostream>
+#include <string>
+
+namespace cppmicroservices {
+namespace service {
+namespace cm {
+namespace test {
+
+TestManagedServiceImpl3::TestManagedServiceImpl3()
+  : m_counter{ 0 }
+{}
+
+TestManagedServiceImpl3::~TestManagedServiceImpl3() = default;
+
+void TestManagedServiceImpl3::Updated(AnyMap const& properties)
```

Comment:
look out for function parameter values which aren't used in the function body. this will cause a build error on linux/mac